src/components/_common/icons/BurgerIcon.tsx

```diff
@@ -0,0 +1,21 @@
+type IconProps = {
+    color: string;
+    fontSize: number;
+  };
```

Comment:
this type is common for all icons. you should eject it to "types.ts" and import for all icons

Reply:
color and size are not required fields. set them as not required and add default values (inherit)